### ISSUE TITLE
Skiplagged scan contains two critical bug, mutually covered each other

### ISSUE THREAD

**Issue by shcheuk** [NECROBOTIO/NecroBot#4741]
Skiplagged return timestamp of seconds. While the code compare it with (DateTime.Now -1970-1-1).milliseconds. Which is 1000 times larger than the expire time. So it basically mark all Pokemon expired already for 50000 years....so why such a big bug not discovered , it is covered by another bug. we should select Pokemon which expire time > now instead of now > expire time . I

**Comment by georgi-huklev** [comment]
I don't even understand what is the point of this check??? expire check must be done before attempting to snipe...
We are discussing this:
https://github.com/NECROBOTIO/NecroBot/blob/master/PoGo.NecroBot.Logic/Tasks/SnipePokemonTask.cs#L345

**Comment by shcheuk** [comment]
Yes, it better move inside  the for each loop containing the snipe(xxx) functions, just like the skiplaggged one. Other wise it may not be expired when u get it from the web, but expired after u caught a few of them . And it is better write like expire time - 5 second > datetime.now , to allow some buffer for the teleport and encounter request ...

**Comment by TheBorna** [comment]
https://github.com/NECROBOTIO/NecroBot/blob/master/PoGo.NecroBot.Logic/Tasks/SnipePokemonTask.cs#L345
![image](https://cloud.githubusercontent.com/assets/18235651/17571399/5327c186-5f50-11e6-9451-3cbfb330692f.png)

You can easily edit it from web if you do not know to setup pull requests from editor..

**Comment by georgi-huklev** [comment]
Yes but I am also testing. Fixing this still did not work for me. THere is something else wrong in the logic.
